app/views/WhatYouOwe.scala.html

```diff
@@ -436,9 +436,9 @@
         @if(!whatYouOweChargesList.isChargesListEmpty) {
             @saNoteParagraph
         }
-        @whatYouOweChargesList.balanceDetails.getAbsoluteUnAllocatedCreditAmount.map { unallocatedCreditAmt =>
-            @if(unallocatedCreditAmt > 0) {
-                @unallocatedCreditParagraphContent(unallocatedCreditAmt.toCurrencyString)
+        @whatYouOweChargesList.balanceDetails.availableCredit.map { availableCreditAmt =>
```

Comment:
same suggestion as per @Daveo-G PR early this week, we should avoid dealing with API structure directly, use what we have already define in the WhatYouOweChargeList 
<img width="1375" alt="image" src="https://github.com/user-attachments/assets/e380708e-3f9c-4dac-9349-5d7b34124f18" />